static/src/javascripts/projects/common/modules/navigation/membership.js

```diff
@@ -10,6 +10,7 @@ import config from 'lib/config';
 import bean from 'bean';
 import arrowRight from 'svgs/icon/arrow-right.svg';
 import type { Banner } from 'common/modules/ui/bannerPicker';
+import { isUserLoggedIn } from 'common/modules/identity/api';
 import userPrefs from 'common/modules/user-prefs';
 
 const accountDataUpdateLink = accountDataUpdateWarningLink => {
```

Comment:
maybe not for this PR but it always confuses me that this banner is just called `membership`. could we make it `membership-action-required.js` so match the `messageCode`?

Reply:
well it shouldn't use the word membership at all since its also for contributions and fairly soon digitalpack too. i've already started creating duplicate events with the mma keyword instead, with the aim of phasing the membership word out in a future PR